src/main/java/com/blibli/future/controller/ShiftController.java

```diff
@@ -0,0 +1,71 @@
+package com.blibli.future.controller;
+
+import com.blibli.future.exception.IdNotFoundException;
+import com.blibli.future.model.Shift;
+import com.blibli.future.service.api.ShiftService;
+import com.blibli.future.valueObject.ShiftVO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class ShiftController {
+
+    private final String BASE_PATH = "/shift";
+    private final String PATH_WITH_ID = BASE_PATH + "/{id}";
```

Comment:
After doing some "comparative study", I think it should be public so the test can reuse the path
`private final String ...` -> `public final String ...`